docs/dev.md

```diff
@@ -64,6 +70,68 @@ dep ensure
 /go/bin/deepcopy-gen --output-base=/go/src --input-dirs=github.com/kubeflow/experimental-kvc/pkg/apis/cr/v1/...
 go build -gcflags "-N -l" github.com/kubeflow/experimental-kvc
 ```
+### Developing on a workstation
```

Comment:
One of the main reason we created the `docker_make` script is to avoid this? Why is it better to follow such complex instructions than just executing `docker_make build`?

Reply:
It's up to you guys @balajismaniam and @Ajay191191 
Feel free to close this PR.
My thinking was `docker_make` is just too slow and I don't use it.
I thought it'll be nice to share the knowledge, but maybe you guys don't think it belongs here and I'm OK with it.

Thanks.

Reply:
Makes sense @ashahba. After you address @Ajay191191's comment we can merge it. Thanks.

Reply:
@Ajay191191 I can remove that line, since user has already cloned the code 🙂